test/tests/parallel/set3/mta/smtp_delivery.bats

```diff
@@ -102,50 +102,43 @@ function setup_file() {
 }
 
 @test "should successfully authenticate with good password (plain)" {
-  _run_in_container_bash 'nc -w 5 0.0.0.0 25 < /tmp/docker-mailserver-test/auth/smtp-auth-plain.txt'
-  assert_success
+  _send_email 'auth/smtp-auth-plain' '' '-w 5 0.0.0.0 25'
```

Comment:
I can't say that I'm fond of the empty arg approach, it's a "code smell"? (_sadly this is more of an issue with bash and how annoying it is to make more flexible with params_)

Only two calls in the Fail2Ban test need to provide an explicit alternative container name to run the command in. It seems that would be more suitable to have the container name arg last? (_especially since the IP is changed anyway, so all args are present_)

Reply:
A good idea; in an earlier version I had the rest of all parameters appended (didn't work too well though), so I was stuck in thinking the name of the container belongs there. If we need to provide the container name in the future more often though, we will be replicating `0.0.0.0 25`, but for the time being, you are correct, I will change that.

Reply:
You can also provide the container name implicitly (prior to the method call), like the command to run in containers bash being called internally uses. But for now swapping the two params should suffice I think.

Reply:
Inward thinking about this too. I will have a look again.

Reply:
Done in 6a3da54b0c7511ace4389b7fcead6269fea46310